docs/HowTo/Configure/Consensus-Protocols/QBFT.md

```diff
@@ -0,0 +1,119 @@
+---
+description: Configuring QBFT consensus
+---
+
+# Configuring QBFT consensus
+
+GoQuorum implements the QBFT Proof-of-Authority (PoA) consensus protocol.
+You can [create a private network using QBFT](../../../Tutorials/Private-Network/Create-QBFT-Network.md).
+
+!!! warning
+
+    QBFT is currently an early access feature. It is not recommended for production networks with business critical impact.
```

Comment:
Do we by any chance have an estimated date that QBFT becomes generally available?

Reply:
putting myself in the thread because I also want to know when this happens (for EEA who are watching for it)

Reply:
@joshuafernandes after this release and we see the feedback from users that have upgraded.